package.json

```diff
@@ -24,34 +24,35 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^5.1.7",
-    "ember-cli-deploy-plugin": "^0.2.9"
+    "ember-cli-babel": "^6.0.0",
+    "ember-cli-deploy-plugin": "^0.2.9",
+    "path": "^0.12.7",
+    "request": "^2.81.0",
+    "request-promise": "^4.2.1",
+    "rsvp": "^3.5.0"
```

Comment:
Yup I just removed it because that didn't fix the issue either. I think all node environments have `fs` since it's part of node's core, so we don't need to install it? Anyway, that's what we're trying here.